colibre/auto_plotter/depletion_relations.yml

```diff
@@ -1,7 +1,7 @@
 oxygen_abundance_v_dust_to_gas:
   type: "scatter"
   legend_loc: "lower right"
-  selection_mask: "derived_quantities.dust_valid_abundances_100_kpc"
+  selection_mask: "derived_quantities.is_active_30_kpc"
```

Comment:
@EvgeniiChaikin this eliminates the 'high' [O/H] values

Reply:
Nice! 

- Do we want `"derived_quantities.is_active_30_kpc"` or `"derived_quantities.is_active_100_kpc"`?
- Since `"dust_valid_abundances_100_kpc"` is no longer used, do we want to keep it in the registration file?